.github/workflows/ut_ci_workflow.yml

```diff
@@ -4,16 +4,21 @@
 name: CI Unit tests
 
 on:
+  schedule:
+    # Monday to Thursday 1 AM PDT build
+    # * is a special character in YAML so you have to quote this string
+    - cron: "0 8 * * 1,2,3,4"
   push:
-    branches: [ dev, master, main ]
+    branches: [ dev, master, main, release/* ]
   pull_request:
-    branches: [ dev, master, main ]
+    branches: [ dev, master, main, release/* ]
 
 jobs:
   build:
     name: "Python UT CI Run"
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: true
```

Comment:
Making the fail-fast explicit. The other tests have `fail-fast: false` to run them completely.